onebusaway-android/src/main/java/org/onebusaway/android/ui/PreferencesActivity.java

```diff
@@ -121,6 +126,11 @@ public class PreferencesActivity extends PreferenceActivity
 
     ListPreference mThemePref;
 
+
```

Comment:
please watch your newlines in between instance variables. one blank newline is fine; two looks strange.